### DISCUSSION TITLE
am wondering how to get the full event object when using RadioGroup in chakra-ui-next

### DISCUSSION THREAD

**Discussion by devclouddz** [chakra-ui/chakra-ui#2423] [Q&A] [answered]
hello guys am stuck in a little problem, am using RadioGroup in react and my problem is that i cant get the event object instead i get only the value, in my case i want to get the name and the value when the onChange event fire

```
 <RadioGroup name="distance" onChange={handleChange}>
     <Radio value="0.5">0.5</Radio>
     <Radio value="2000">1</Radio>
     <Radio value="3000">2</Radio>
     <Radio value="4000">3</Radio>
     <Radio value="5000">4</Radio>
 </RadioGroup>
```
```
 const handleChange = (e) => {
    console.log(e);//getting only the value
   
  };
```

**Comment by willem-aart** [comment, marked as answer]
+1 — Just ran into a use case where having the full event object would make life easier.

As a workaround, you could define handlers on the individual `Radio` components and retrieve the event object from there.

**Reply by devclouddz** [reply]
thanks for answering me , you are awesome
